CourseGrab/AppDelegate.swift

```diff
@@ -40,7 +40,7 @@ extension AppDelegate: GIDSignInDelegate {
 
     func sign(_ signIn: GIDSignIn!, didSignInFor user: GIDGoogleUser!, withError error: Error!) {
         if let error = error {
-            print(error.localizedDescription)
+            print(error)
```

Comment:
We should remove this print statement and build UI to handle the error. I'll make this a new issue.